src/worktree.c

```diff
@@ -455,24 +460,16 @@ int git_worktree_is_locked(git_buf *reason, const git_worktree *wt)
 	return ret;
 }
 
-int git_worktree_name(const git_worktree *wt, git_buf *name)
+const char * git_worktree_name(const git_worktree *wt)
```

Comment:
Nitpick: I think the style guide would say "left padded * operators", so `type *func(...)`.